test/images/image-util.sh

```diff
@@ -186,8 +188,7 @@ build() {
     docker buildx build --progress=plain --no-cache --pull --output=type="${output_type}" --platform "${os_name}/${arch}" \
         --build-arg BASEIMAGE="${base_image}" --build-arg REGISTRY="${REGISTRY}" --build-arg OS_VERSION="${os_version}" \
         -t "${REGISTRY}/${image}:${TAG}-${suffix}" -f "${dockerfile_name}" \
-	--label "image_version=${TAG}" --label "commit_id=${GIT_COMMIT_ID}" \
-	--label "git_url=https://github.com/kubernetes/kubernetes/tree/${GIT_COMMIT_ID}/test/images/${img_folder}" .
+	--label "image_version=${TAG}" --label "git_tag=${GIT_TAG}" .
```

Comment:
I also see `TAG` here but in the cloudbuild.yaml you have replaced the `TAG` with `GIT_TAG`, wondering if that creates any issue?

Reply:
It seems that ``PULL_BASE_SHA`` is more commonly used in other jobs too, and it's mentioned here as well: https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#job-environment-variables

Went ahead with ``PULL_BASE_SHA`` instead.